clients/cobol-lsp-vscode-extension/src/services/CopybookFix.ts

```diff
@@ -26,7 +26,7 @@ export class CopybookFix {
         if (options.hasPaths) {
             actions.push(downloadCopybookAction);
         }
-        if (reasonMsg !== "Configuration was updated") {
+        if (reasonMsg !== "Configuration updated. Please provide correct settings for nested copybooks and fetch again.") {
```

Comment:
you have updated one string in a condition which now always return true so entry this condition, you have to update the source of the `reasonMsg` which is in the `extension.ts`

Reply:
I do believe that the message is wrong in essence, if you just updated the settings why would you like to update with correct settings again, are you making sure on the technical way that in order to throw this message the user has provided the wrong configuration?

Reply:
Thanks, changed.